openhcl/hcl/src/ioctl.rs

```diff
@@ -1611,7 +1610,7 @@ pub struct ProcessorRunner<'a, T> {
     sidecar: Option<SidecarVp<'a>>,
     _no_send: PhantomData<*const u8>, // prevent Send/Sync
```

Comment:
You can leave this, but also OK to remove, since `UnsafeCell` is neither `Send` nor `Sync`.